components/cloud-event-gateway-proxy/cmd/cloud-event-gateway-proxy/main.go

```diff
@@ -31,10 +31,7 @@ func main() {
 	defer client.CloseIdleConnections()
 
 	// configure message sender
-	messageSender, err := sender.NewHttpMessageSender(cfg.EmsPublishURL, client)
-	if err != nil {
-		logger.Fatalf("Unable to create message sender with error: %s", err)
-	}
+	messageSender := sender.NewHttpMessageSender(cfg.EmsPublishURL, client)
```

Comment:
Can you explain why we are ignoring the error here? 🤔

Reply:
The initial implementation of the `NewHttpMessageSender` function was always returning a `nil` error, which was not needed. So it is simplified to the implementation below:

https://github.com/kyma-project/kyma/blob/0aaf7e9e3e2d520def70de97930645d3cf646a80/components/cloud-event-gateway-proxy/pkg/sender/sender.go#L14-L17

Reply:
Ah, I missed the change in the signature.